bm-timeline/src/Adoption.hs

```diff
@@ -0,0 +1,220 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main
+  (
+    main
+  )
+where
+
+import Control.Monad (forM)
+import qualified Data.HashMap.Strict as HM
```

Comment:
I thought that we align imports like this:

```
import           Control.Monad (forM)
import qualified Data.HashMap.Strict as HM
```

Reply:
not necessarily; we might do a cleanup at some point